README.md

```diff
@@ -1,43 +1,51 @@
-homebrew-neovim
-===============
+# homebrew-neovim
+
 [![Build Status](https://travis-ci.org/neovim/homebrew-neovim.svg?branch=master)](https://travis-ci.org/neovim/homebrew-neovim)
 
-Homebrew formula for neovim.
+Homebrew formula for Neovim.
 
-## Install (stable)
+## Installation
 
-To install the latest stable release:
+Since this formula isn't part of Homebrew, add it as tap first:
 
-    $ brew install neovim/neovim/neovim
+    $ brew tap neovim/neovim
 
-To **upgrade** from a previous version:
+To install the latest stable release:
```

Comment:
I divided "Installation" and "Upgrading" into two sections. It can't get any clearer. :-P

(And between you and me, I kind of expect every Homebrew user to know when to use `install`, `update`, and `upgrade` anyway.. They're the absolute basic commands.)